jujugui/static/gui/src/app/components/boolean-config/_boolean-config.scss

```diff
@@ -4,13 +4,18 @@
     position: relative;
 
     &--title {
-      padding-top: 8px;
+        margin: auto;
+        margin-left: 0;
```

Comment:
This would be better as `margin: auto auto auto 0` or are you going for `margin: auto 0;`?